rafthttp/pipeline.go

```diff
@@ -98,11 +97,11 @@ func (p *pipeline) handle() {
 			reportSentFailure(pipelineMsg, m)
 
 			if p.errored == nil || p.errored.Error() != err.Error() {
-				log.Printf("pipeline: error posting to %s: %v", p.to, err)
+				plog.Infof("error posting to %s (%v)", p.to, err)
```

Comment:
I thought disconnection is a common case of failure, but it is still a failure. will change.